includes/class-sensei-course.php

```diff
@@ -2902,10 +2904,8 @@ public static function load_single_course_lessons_query() {
 
 		}
 
-		// phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited -- Used for lesson loop on single course page. Reset in hook to `sensei_single_course_lessons_after`.
-		$wp_query = new WP_Query( $course_lesson_query_args );
-
-	}//end load_single_course_lessons_query()
+		return new WP_Query( $course_lesson_query_args );
+	}//end get_single_course_lessons_query()
```

Comment:
I find these comments bad practice, do you agree?

Reply:
Totally! 😅
Updated for this file here: https://github.com/Automattic/sensei/pull/3032/commits/10b205e2e4e7890f0d1235fbc4f07d82e30083b6